.env.template

```diff
@@ -1 +1,2 @@
 WIX_CLIENT_ID=35a15d20-4732-4bb8-a8ef-194fd1166827
+SESSION_COOKIE_SIGNING_SECRET=SOME_SECRET
```

Comment:
is this secret set by the wix-headless feature when we connect to wx?

Reply:
it has completely nothing to do with wix headless